### ISSUE TITLE
source code documentation not build properly on RTD

### ISSUE THREAD

**Issue by japaf** [japaf/foamgen#15]
There is no problem when building locally.

**Comment by japaf** [comment]
Build passes but returns following for each auto-generated file

```
WARNING: [autosummary] failed to import 'foamgen.generation': no module named foamgen.generation
Failed to import 'foamgen.generation': no module named foamgen.generation
```

**Comment by japaf** [comment]
Could be connected to PYTHONPATH (see [here](https://github.com/sphinx-doc/sphinx/issues/4724)).

**Comment by japaf** [comment]
Resolved in 8b5c1ee5f3e28cb2f7a396f7f60dc972439b44a7. I had to add pandas to mock imports.
